nautobot/extras/tests/example_jobs/api_test_job.py

```diff
@@ -17,10 +22,10 @@ class Meta:
     var4 = ObjectVar(model=Role)
 
     def run(self, var1, var2, var3, var4):
-        self.log_debug(message=var1)
-        self.log_info(message=var2)
-        self.log_success(message=var3)
-        self.log_warning(message=var4)
+        logger.debug(var1)
+        logger.info(var2)
+        logger.info(var3)
+        logger.warning(var4)
```

Comment:
Not that we want to document this too heavily but for external calls and testing, we should be able to call functions in the job to test locally, and log messages won't :boom: because of a failure to access the DB. Thinking of testing functions that call to external systems without having to run the whole job.

Reply:
If the job isn't running on a celery worker or using `JobResult.execute_job`, the database logger handler won't be present. Directly running any code that sends messages to the celery task logger should work.